src/core/unicode.js

```diff
@@ -1629,13 +1628,13 @@ var getNormalizedUnicodes = getArrayLookupTableFactory(function () {
 });
 
 function reverseIfRtl(chars) {
-  var charsLength = chars.length;
+  const charsLength = chars.length;
   // Reverse an arabic ligature.
   if (charsLength <= 1 || !isRTLRangeFor(chars.charCodeAt(0))) {
     return chars;
   }
-  var s = "";
-  for (var ii = charsLength - 1; ii >= 0; ii--) {
+  let s = "";
+  for (let ii = charsLength - 1; ii >= 0; ii--) {
     s += chars[ii];
   }
   return s;
```

Comment:
*Nit:* Can we please change this to use a buffer, rather than string concatenation, since that should be more efficient for short strings (which is what we're dealing with here). 

*Obviously making these changes in a separate commit would be fine.*

Reply:
Done in a new commit. Good idea!